packages/@okta/vuepress-theme-prose/components/HeaderBanner.vue

```diff
@@ -98,6 +98,10 @@ body.dark-theme .header-banner {
   font-weight: 400;
   font-size: 14px;
 
+  p {
+    font-size: 14px;
```

Comment:
nit: is this required? we have the same font size set for parent element

Reply:
Yes it is required, because `p` element in general has a `font-size: 100%`, and 100% is equivalent to 16px in dev docs. That was overriding the parent font-size. Hence, we had to add this font-size specifically here for the banner.